addons/website/static/src/js/content/menu.js

```diff
@@ -141,7 +141,7 @@ const BaseAnimatedHeader = animations.Animation.extend({
      * @private
      */
     _updateMainPaddingTop: function () {
-        this.headerHeight = this.$el.outerHeight();
+        this.headerHeight = this.$el[0].classList.contains('o_header_sidebar') ? 0 : this.$el.outerHeight();
```

Comment:
Seems legit :)

Note: `this.el` is the same as `this.$el[0]`, you could use that :+1:

Reply:
Done :)